File: python/calculate_mode.py
def calculate_mode(lst):
    answer =[]
    freq = 0
    my_dict = {}
    for i in lst:
        times = lst.count(i)
        my_dict[i] = times
        if times > freq:
            freq = times

    for i in my_dict:
        if my_dict[i] == freq:
            answer.append(i)
    return answer 

File: js/calculateMode.js
exports.calculateMode = function(arr) {
    const answer = []
    let freq = 0
    let obj = {}
    
    for (let i = 0; i < arr.length; i++) {
        const item = arr[i];
        obj[item] = (obj[item] || 0) + 1;
        if (obj[item] > freq) {
            freq = obj[item];
        }
    }

    for (const key in obj) {
        if (obj[key] === freq) {
            answer.push(key);
        }
    }
    return answer
}
